controller.py

```diff
@@ -11,19 +10,42 @@
 def login():
     usuario = web_usuario(**request.json)
     if valida_usuario(**usuario):
-        post_usuarios(**usuario)
-        user = get_usuarios(usuario['nome_completo'])
+        id_usuario = post_usuarios(**usuario)
+        user = get_usuarios(id_usuario)
         return jsonify(usuario_db(user))
     else:
-        return jsonify({"erro":"Usuário Inválido"})
+        return jsonify({"erro":"Este usuario não pode ser cadastrado"})
+
+@app.route("/usuarios", methods=['GET'])
+def checar_usuario():
+    nomeUsuario = nome_web(**request.args)
+    usuarios = mostrar_usuarios(nomeUsuario)
```

Comment:
```suggestion
    nome_usuario = nome_web(**request.args)
    usuarios = mostrar_usuarios(nome_usuario)
```
No python normalmente usamos _ no lugar de camel case pra variaveis